app/src/main/res/layout/dashboard_tools_fragment.xml

```diff
@@ -33,8 +32,10 @@
                 android:name="org.cru.godtools.ui.dashboard.tools.ToolsCategoryFragment"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
+                tools:layout_height="match_parent"
+                android:tag="ToolsCategory"
```

Comment:
what's your reason for setting a view tag?

Reply:
@frett I thought it was best practice